primeqa/qg/run_qg.py

```diff
@@ -74,13 +74,13 @@ class DataTrainingArguments:
             "help": "Config of the dataset loaded, e.g. 'secondary_task' for TyDiQA"
         },
     )
-    train_file_path: Optional[str] = field(
-        default="train_data.pt",
-        metadata={"help": "Path for cached train dataset"},
+    train_file: Optional[str] = field(
+        default=None,
+        metadata={"help": "local file(s) in .json to train on."},
     )
-    valid_file_path: Optional[str] = field(
-        default="valid_data.pt",
-        metadata={"help": "Path for cached valid dataset"},
+    eval_file: Optional[str] = field(
+        default=None,
+        metadata={"help": "local file(s) in .json to evaluate on."},
```

Comment:
local file(s) in .json to evaluate on.
```suggestion
        metadata={"help": "local file(s) in .jsonl to evaluate on."},
```